README.md

```diff
@@ -74,6 +74,13 @@ Rockstar uses a very similar type system to that defined by the [ECMAScript type
 * **Number** - Numbers in Rockstar are stored using the [DEC64](http://www.dec64.com/) numeric type.
 * **String** - Rockstar strings are sequences of 16-bit unsigned integer values representing UTF-16 code units.
 * **Object** - a collection of named data properties, as in ECMAScript.
+
+### Literal Words vs Reserved Keywords
+
+Words that are used to construct a literal of a certain type are referred to as "Literal Words" and words that are used to construct various syntax constructs are referred to as "Reserved Keywords".
+
+- Literal Words: `mysterious`, `null`, `nothing`, `nowhere`, `nobody`, `empty`, `gone`, `true`, `right`, `yes`, `ok`, `false`, `wrong`, `no`, `lies`
```

Comment:
`maybe` and `definitely maybe` are also reserved and should probably be included in this list.

Reply:
I got the impression they would fall under the category of "Reserved Keywords" as opposed to "Literal Words"

Reply:
They are, but they appear to be intended to function as literal words or modifiers when they are unreserved. My concern is something like this:

```
Tommy was maybe true
```

which currently evaluates to `54` but if `maybe` were unreserved then this would change the meaning of the statement. To avoid problems I think this statement should be defined as an error.

Reply:
Fair enough, I think reserving them would be good for now. I'll update it after lunch.